local_costmap:
  global_frame: map
  robot_base_frame: base_footprint
  update_frequency: 20.0       #Cập nhật map  (Hz)
  publish_frequency: 20.0      #Hiển thị thông tin (Hz)
  static_map: false
  rolling_window: false       #Sử dụng cuộn (Phải để  false nêu static_map là true)
  always_send_full_costmap: true
  width: 10.0                #Chiều rộng của bản đồ  (m)
  height: 10.0               #Chiều dài của bản đồ (m)
  resolution: 0.05           #Độ phân giải của map (mét/ô)
  origin_x: 0.0              #Vị trí ban đầu so với global_frame
  origin_y: 0.0              #Vị trí ban đầu so với global_frame
  tranform_tolerance: 1    #Độ trễ  trong dữ liệu biến đôi (tf) có thể chấp nhận được (giữa global_frame và robot_base_frame). Nếu vượt qúa thời gian sẽ ngừng robot
  planner_frequency: 5
  planner_patiente: 5.0
  transform_tolerance: 1.0


  plugins:
    #- {name: obstacle_3d_layer,       type: "costmap_2d::VoxelLayer"}
    #- {name: static_layer,            type: "costmap_2d::StaticLayer"}
    - {name: obstacle_2d_layer,       type: "costmap_2d::ObstacleLayer"}
    - {name: inflater_layer,          type: "costmap_2d::InflationLayer"}